lib/valueflow.cpp

```diff
@@ -4508,7 +4506,6 @@ static void valueFlowLifetime(TokenList *tokenlist, SymbolDatabase*, ErrorLogger
             auto captureVariable = [&](const Token* tok2, LifetimeCapture c, std::function<bool(const Token*)> pred) {
                 if (varids.count(tok->varId()) > 0)
                     return;
-                ErrorPath errorPath;
```

Comment:
I assume this is https://trac.cppcheck.net/ticket/9823 since the unused variable is inside a lamdba.

Reply:
Turns out there's a different issue related to having lamdba variables: https://trac.cppcheck.net/ticket/11215